packages/flash-calendar/src/components/CalendarList.stories.tsx

```diff
@@ -112,6 +112,14 @@ export function ImperativeScrolling() {
 
   const ref = useRef<CalendarListRef>(null);
 
+  const onCalendarDayPress = useCallback((dateId: string) => {
+    loggingHandler("onCalendarDayPress");
+
+    const additionalOffset = -2; // Offset to account for selected day paddingTop
```

Comment:
Style on today's button gets cropped a bit on top, so we can leverage the `additionalOffset` param here to add a bit of a padding. 
I'm not sure if I missed a space property on the `scrollToWeek` function though, that would handle this and make `additionalOffset` for this calendar unnecessary, either way I believe this prop is useful so users can adjust things at their will.

Reply:
I couldn't tell where the offset issue comes from either. I'm fine with the additionalOffset since once thing I realized while building Flash Calendar is how hard it is to actually ensure everything aligns perfectly. Most people won't have the patience to diligently fix these small issues at their custom calendars - it's nice to offer an escape hatch with `additionalOffset`